src/proposals/20250807/SparkEthereum_20250807.sol

```diff
@@ -0,0 +1,136 @@
+
+    function newListings() public pure override returns (IEngine.Listing[] memory) {
+        IEngine.Listing[] memory listings = new IEngine.Listing[](1);
+
+        listings[0] = IEngine.Listing({
+            asset:              PYUSD,
+            assetSymbol:        'PYUSD',
+            priceFeed:          PYUSD_PRICE_FEED,
+            rateStrategyParams: Rates.RateStrategyParams({
+                optimalUsageRatio:             _bpsToRay(0),
+                baseVariableBorrowRate:        0,
+                variableRateSlope1:            _bpsToRay(0),
+                variableRateSlope2:            _bpsToRay(0),
+                stableRateSlope1:              0,
+                stableRateSlope2:              0,
+                baseStableRateOffset:          0,
+                stableRateExcessOffset:        0,
+                optimalStableToTotalDebtRatio: 0
+            }),
+            enabledToBorrow:       EngineFlags.ENABLED,
+            stableRateModeEnabled: EngineFlags.DISABLED,
+            borrowableInIsolation: EngineFlags.DISABLED,
+            withSiloedBorrowing:   EngineFlags.DISABLED,
+            flashloanable:         EngineFlags.ENABLED,
+            ltv:                   0,
+            liqThreshold:          0,
+            liqBonus:              10_00,
+            reserveFactor:         10_00,
+            supplyCap:             50_000_000,
+            borrowCap:             25_000_000,
+            debtCeiling:           0,
+            liqProtocolFee:        10_00,
+            eModeCategory:         2
+        });
+
+        return listings;
+    }
+
+    function _postExecute() internal override {
+        LISTING_ENGINE.POOL_CONFIGURATOR().setReserveInterestRateStrategyAddress(Ethereum.USDS, USDS_DAI_IRM);
+        LISTING_ENGINE.POOL_CONFIGURATOR().setReserveInterestRateStrategyAddress(Ethereum.DAI,  USDS_DAI_IRM);
+        LISTING_ENGINE.POOL_CONFIGURATOR().setReserveInterestRateStrategyAddress(Ethereum.USDC, USDC_USDT_PYUSD_IRM);
+        LISTING_ENGINE.POOL_CONFIGURATOR().setReserveInterestRateStrategyAddress(Ethereum.USDT, USDC_USDT_PYUSD_IRM);
+        LISTING_ENGINE.POOL_CONFIGURATOR().setReserveInterestRateStrategyAddress(PYUSD,         USDC_USDT_PYUSD_IRM);
+
+        ICapAutomator capAutomator = ICapAutomator(Ethereum.CAP_AUTOMATOR);
+
+        capAutomator.setSupplyCapConfig({ asset: PYUSD, max: 500_000_000, gap: 50_000_000, increaseCooldown: 12 hours });
+        capAutomator.setBorrowCapConfig({ asset: PYUSD, max: 475_000_000, gap: 25_000_000, increaseCooldown: 12 hours });
+
+        // Seed the new pyUSD pool
+        IERC20(PYUSD).approve(address(LISTING_ENGINE.POOL()), 1e6);
+        LISTING_ENGINE.POOL().supply(PYUSD, 1e6, address(this), 0);
+
+        IPoolConfigurator(Ethereum.POOL_CONFIGURATOR).setLiquidationProtocolFee(PYUSD, 10_00);
+        IPoolConfigurator(Ethereum.POOL_CONFIGURATOR).setReserveFlashLoaning(Ethereum.USDS, true);
+
+        RateLimitHelpers.setRateLimitData(
```

Comment:
Actually this should use `onboardAaveToken` instead

Reply:
yeah changed it to use onboardAaveToken